cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -136,6 +136,9 @@
   private static final String STOPPED = "STOPPED";
   private static final String RUNNING = "RUNNING";
 
+//  @Rule // 120 seconds max per method tested
```

Comment:
remove?

Reply:
oh yeah that's a relic of something I tested out. will remove.